proxy_test.go

```diff
@@ -276,7 +276,7 @@ func TestProxyServeStopHTTP(t *testing.T) {
 	}()
 
 	// Stop the HTTP server only, causing Serve to exit.
-	waitForHTTPStart(t, &p, 3*time.Second)
```

Comment:
I'm confused about how this ever worked, the second argument to waitForHTTPStart is not a pointer.

Reply:
The type of `p` has changed from being an object to being a pointer to the object.

Reply:
I see, my bad.